### ISSUE TITLE
Sevice is triggered even if the bot rerun command is quoted

### ISSUE THREAD

**Issue by chrisburr** [conda-forge/conda-forge-webservices#411]
In https://github.com/conda-forge/flit-feedstock/pull/23#issuecomment-708164116 I wrote a message containing "(`@conda-forge-admin, please rerun bot`)" but the bot was still triggered despite the code tags.

**Comment by beckermr** [comment]
Yup. This is a known thing. We get the raw text in the heroku server. You have to quote part in code tags and part in ticks.
